components/esp_lvgl_port/esp_lvgl_port.c

```diff
@@ -56,6 +56,24 @@ typedef struct {
 } lvgl_port_touch_ctx_t;
 #endif
 
+#if __has_include ("iot_button.h")
+
+typedef enum {
+    LVGL_PORT_NAV_BTN_PREV,
+    LVGL_PORT_NAV_BTN_NEXT,
+    LVGL_PORT_NAV_BTN_ENTER,
+    LVGL_PORT_NAV_BTN_CNT,
+} lvgl_port_nav_btns_t;
```

Comment:
I don't really understand this. What if I need more or less than exactly 3 buttons?

Reply:
These are only navigation buttons. How can you use more than these buttons for navigation? LVGL support only these three buttons for navigation (simulate encoder using). I have an example (modified ESP-BOX examples), but it is using new esp-box-lite, which is not done.

Reply:
I didn't know much about LVGL's navigation. All clear now, thanks!